test/test_limbo.py

```diff
@@ -102,6 +102,39 @@ def test_handle_message_ignores_slackbot():
     event = {"user": "USLACKBOT"}
     assert limbo.handle_message(event, server) == None
 
+def test_handle_message_ignores_nomention():
+    msg = u"!echo Iñtërnâtiônàlizætiøn bot"
```

Comment:
Tests using non-ASCII characters are great!

Reply:
I can't take credit for this: The existing test-cases were this way.  And in fact, the non-ASCII test cases caught a bug in the Ubuntu Dockerfile.test -- the locale-related pieces of that file were put in to make the non-ASCII cases pass.